Bill3.java

```diff
@@ -1,8 +1,16 @@
 // Splits a restaurant bill evenly among three diners.
 public class Bill3 {
 	public static void main(String[] args) {
-		// To get you started, here is the first line in the program:
+		//names of diners
 	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+		String name2 = args[1];
+		String name3 = args[2];
+		int cost = Integer.parseInt(args[3]); //full amount needed to pay
+
+		double splitAmount = Math.ceil((double)cost/3);
```

Comment:
Spaces between operations!
It should have looked like: "double splitAmount = Math.ceil((double) cost / 3);".
‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎ ‎↑‎ ‎ ‎ ‎ ↑↑